CHANGELOG.md

```diff
@@ -1,11 +1,28 @@
-<a name="v1.3.0"></a>
-# [v1.3.0 🌈](https://github.com/honeycombio/terraform-aws-integrations/releases/tag/v1.3.0) - 18 Apr 2024
+<a id="v1.3.1"></a>
+# [v1.3.1 🌈](https://github.com/honeycombio/terraform-aws-integrations/releases/tag/v1.3.1) - 2024-04-24
```

Comment:
the date needs to be updated?

Reply:
the version of `changelog-from-release` https://github.com/rhysd/changelog-from-release/releases/tag/v3.8.0 updates the date to be the standard ISO format

Reply:
Ooh I think it auto-generated the date based on the date of the draft release. The draft release was created on April 24. If we change that date to today then re-running this should use today's date.

Reply:
Oh I don't know if it's possible. Weird. Maybe delete that release and create a new one?

Reply:
fixed now